components/releases/ReleaseSection.tsx

```diff
@@ -0,0 +1,107 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import Release from "@/app/api/releases/route";
+
+const ReleaseSection: React.FC = () => {
+  const [releases, setReleases] = useState<Release[]>([]);
+  useEffect(() => {
+    fetch("/api/releases")
+      .then((res) => res.json())
+      .then((data: Release[]) => setReleases(data))
+      .catch((error) => console.error("Error fetching releases:", error));
+  }, []);
+
+  if (releases.length === 0) {
+    return (
+      <>
+        <div className="w-full h-10 bg-gray-200 dark:bg-gray-700 animate-pulse rounded" />
+        <div className="w-full h-10 bg-gray-200 dark:bg-gray-700 animate-pulse rounded" />
+        <div className="w-full h-10 bg-gray-200 dark:bg-gray-700 animate-pulse rounded" />
+        <div className="w-full h-10 bg-gray-200 dark:bg-gray-700 animate-pulse rounded" />
+      </>
+    );
+  }
```

Comment:
Make this an SSR component instead of a client component. See how it's done in active projects.tsx

Reply:
made the changes, please have a look!